sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -1,9 +1,31 @@
+namespace Azure.Core
+{
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MonitorQueryDateTimeRange : System.IEquatable<Azure.Core.MonitorQueryDateTimeRange>
```

Comment:
This name is pretty long.  Since it's no longer an Azure.Core type, should/could we consider something shorter like `MonitorQueryTimeRange`?  I don't feel strongly though, so I'd recommend pinging @KrzysztofCwalina to get his thoughts and I'll be happy with wherever the two of you land.